Purchases/Public/RCPurchaserInfo.h

```diff
@@ -17,8 +18,11 @@ NS_ASSUME_NONNULL_BEGIN
 NS_SWIFT_NAME(PurchaserInfo)
 @interface RCPurchaserInfo : NSObject
 
+/// Entitlements attached to this purchaser info
+@property (readonly) RCEntitlementInfos *entitlements;
+
 /// All active *entitlements*.
-@property (readonly) NSSet<NSString *> *activeEntitlements;
+@property (readonly) NSSet<NSString *> *activeEntitlements DEPRECATED_MSG_ATTRIBUTE("Check the .isActive property on the EntitlmentInfo object in entitlements instead.");
```

Comment:
We lose a really nice clean check here. 
```[info.activeEntitlements contains:@"pro"]```

Now to check for pro, do you have to loop?
```info.entitlements.active[@"pro"] != nil``` works better than checking each individual is active

This works well: